### DISCUSSION TITLE
[Feature] Focused iOS uploads

### DISCUSSION THREAD

**Discussion by sugarfunk** [immich-app/immich#4108] [Feature Request]
### The feature

I searched for this in issues and discussions, and checked other repositories and don’t think I believe I have seen anyone else bring up this request. 

What would be super cool is to have an iOS app focused mode for the initial downloads or large volume of pictures.

I look to the Synology photos app for reference though, I couldn’t find any documentation to link here.

they offer a “focus mode” for phone uploads. Basically, it is immich’s foreground upload but what it does is prevents the screen from turning off, but also dims the screen to black so that the user can allow large volumes of pics to backup without relying on scheduled background uploads. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by radh21301** [comment]
Something I did: Play a small YouTube video(in loop to reduce data usage) in picture-in-picture mode, and then open immich. This should prevent the screen from turning off.
